jupyterlab_git/git.py

```diff
@@ -282,12 +283,19 @@ async def clone(self, path, repo_url, auth=None):
         :return: response with status code and error message.
         """
         env = os.environ.copy()
+        cmd = ["git", "clone"]
+        if versioning:
+            cmd.append("--depth=1")
+        cmd.append(unquote(repo_url))
+
         if auth:
             if auth.get("cache_credentials"):
                 await self.ensure_credential_helper(path)
             env["GIT_TERMINAL_PROMPT"] = "1"
+            cmd.append("-q")
+            current_content = set(os.listdir(path))
```

Comment:
This should be called in the `if` condition L287

This should solves the remaining test failures.